src/modules/ethereum/transactions/gasPrices/GasPriceObject.ts

```diff
@@ -2,8 +2,11 @@ import type { EIP1559 } from '@zeriontech/transactions';
 import type { EIP1559Base } from './EIP1559';
 
 export interface GasPriceObject {
-  classic?: number | string | null;
-  eip1559?: EIP1559 | null;
+  classic: number | string | null;
+  eip1559: EIP1559 | null;
+  optimistic: {
+    underlying: { classic: number | string | null; eip1559: EIP1559 | null };
+  } | null;
```

Comment:
what's the point of this property if it only contains `classic` and `eip1559` data?

Reply:
This is just a data object from backend. Not sure how it is built

Reply:
As I understand, this should help us to build optimistic fees, and at the same time gas price object doesn't have its own classic or eip info to avoid confusion